### PR TITLE
update(docs): Add notes if running into problems when installing/starting

### PR DESCRIPTION
**Pull request by chilts**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,10 @@ After installation completes visit [127.0.0.1:3030](http://127.0.0.1:3030/). Use
 
 **Use `npm run update` to [fetch the latest changes](_scripts/update_all.sh) to servers**
 
+Note 1 : if npm fails when installing (specifically the fxa-content-server) or a number of servers fail to start, it may be worth trying a `npm install --production` and then a `npm install` to see if a two stage install works. Also, a `ulimit -n 2560` may help (to allow more currently open files) if you're on Linux.
+
```
Comment (jrgm) [inline README.md]: `ulimit -n` also works on osx, but only up to 1024 (default on osx is 256). Recent versions of npm (2.11.x) seem to be using more handles.

Comment (rfk) [inline README.md]: FWIW I lost like two hours of my Friday afternoon to this file-handles thing last week....

Comment (chilts) [inline README.md]: Yeah, @vladikoff told me this one. Funnily enough, I thought this was the whole reason izs wrote https://www.npmjs.com/package/graceful-fs ... perhaps we're doing it wrong! :D

Comment (vladikoff) [inline README.md]: It's a bug in npm, see https://github.com/npm/npm/issues/8166

Comment (vladikoff) [inline README.md]: @chilts so the install script already tries a 2 stage install: https://github.com/mozilla/fxa-local-dev/blob/master/_scripts/install_all.sh#L25 so this note can be about `ulimit`

For later, I'm planning to make the `ulimit` command part of the script, once I have OS X Travis testing working properly.

Comment (chilts) [inline README.md]: Ok, I'll remove the install note. Also, happy to just let this PR rot once you have `ulimit` as part of the script.

Comment (vladikoff) [inline README.md]: @chilts nah, this is good for now! Thank you !!

**Pull request merged**